x/evm/keeper/statedb_test.go

```diff
@@ -392,16 +392,15 @@ func (suite *KeeperTestSuite) TestCommittedState() {
 
 	suite.app.EvmKeeper.SetState(suite.address, key, value1)
 
-	commit := suite.app.EvmKeeper.BeginCachedContext()
+	suite.app.EvmKeeper.Snapshot()
 
 	suite.app.EvmKeeper.SetState(suite.address, key, value2)
 	tmp := suite.app.EvmKeeper.GetState(suite.address, key)
 	suite.Require().Equal(value2, tmp)
 	tmp = suite.app.EvmKeeper.GetCommittedState(suite.address, key)
 	suite.Require().Equal(value1, tmp)
 
-	commit()
-	suite.app.EvmKeeper.EndCachedContext()
+	suite.app.EvmKeeper.CommitCacheContexts()
```

Comment:
we should test the `RevertToSnapshot` logic too

Reply:
Done that in the `TestSnapshot` test case now.